test/runtests.jl

```diff
@@ -86,6 +86,10 @@ using Test
 
     val = [39000, 30000, 21000, 37000, 46000]
     @test isnan(mirr(val, 0.10, 0.12))
+    
+    @info "Testing reff"
+    @test round(reff(0.08, 2, 1), digits = 2) == 8.16
+    @test round(reff(0.12, 4, 1), digits = 2) == 12.55
```

Comment:
We can remove one of these cases since they are testing the same logic and having more tests does not increase coverage.